core/src/main/python/wlsdeploy/util/variables.py

```diff
@@ -243,9 +246,18 @@ def _read_value_from_file(file_path):
         line = file_reader.readLine()
         file_reader.close()
     except IOException, e:
-        ex = exception_helper.create_variable_exception('WLSDPLY-01733', file_path, e.getLocalizedMessage(), error=e)
-        _logger.throwing(ex, class_name=_class_name, method_name=method_name)
-        raise ex
+        if model_context.get_validation_method() == 'strict':
+            if validation_result:
+                validation_result.add_error('WLSDPLY-01733', file_path, e.getLocalizedMessage())
+            ex = exception_helper.create_variable_exception('WLSDPLY-01733', file_path, e.getLocalizedMessage(), error=e)
+            _logger.throwing(ex, class_name=_class_name, method_name=method_name)
+            raise ex
+        else:
+            if validation_result:
+                validation_result.add_warning('WLSDPLY-01733', file_path, e.getLocalizedMessage())
```

Comment:
Since missing files are fully expected, can this be an Info instead of a Warning?  Warnings can be distracting as they imply there's something that should be investigated...

Reply:
No, the tool is not only used by the k8s and imagetool.  It can be used on premise and other situation

Reply:
Then add a third state or check for env var?  It's distracting to get spurious warnings.

Reply:
There's a new PR for this change, and a discussion of this topic.
https://github.com/oracle/weblogic-deploy-tooling/pull/443